src/packages/excalidraw/index.tsx

```diff
@@ -243,3 +243,6 @@ export { Button } from "../../components/Button";
 export { Footer };
 export { MainMenu };
 export { useDevice } from "../../components/App";
+export { LiveCollaboration };
+
+export const icons = { usersIcon };
```

Comment:
fix casing (not a component) and namespaced. 

https://github.com/excalidraw/excalidraw/issues/6106